packages/README.md

```diff
@@ -0,0 +1,46 @@
+# Packages
+
+This directory contains two types of packages. Management Packages and Standalone Packages
+
+## Management Packages
+
+This directory contains some carvel packages for management components. Any package that performs the role of managing and operating a workload cluster is a Management package. This management packages are purpose-built to be installed on the management cluster. More details about packages can be found [here](../docs/packages/definitions.md)
+
+This management packages are associated with each in terms of [Meta Package](../docs/packages/definitions.md#meta-package). A Meta Package contains PackageInstall CRs, Secrets and ServiceAccounts, RBAC rules that are needed to install other packages and when the meta package is installed, it installs the other packages.
+
+Below is the hierarchy of management packages association:
+
+- tkg (TKG Meta Package)
+  - framework (Framework Meta Package)
+    - addons-manager
+    - tkr-service
+    - featuregates
+    - tanzu-auth
+    - cliplugins
+  - tkg-clusterclass (TKG ClusterClass Meta Package)
+    - tkg-clusterclass-docker
+    - tkg-clusterclass-aws
+    - tkg-clusterclass-azure
+    - tkg-clusterclass-vsphere
+  - core-management-plugins
+  - tkr-source-controller
+
+As part of the Bootstrap and Management Cluster creation, `management-cluster` plugin installs `tkg` meta package which installs all child packages. As part of this package installation `management-cluster` plugin creates [values.yaml](./tkg/bundle/config/values.yaml) file based on the user input and the configuration is propogated to child packages based on overlays written within the package.
+
+There are few management packages created under this directory but not getting used at the moment and not intgrated with `tkg` meta package. Below is the list of these packages:
```

Comment:
"but not getting used at the moment" -> "which are not used at the moment"
"intgrated with `tkg`" -> "integrated with the `tkg`" (typo and extra "the")